app/components/rating/rating.twig

```diff
@@ -0,0 +1,7 @@
+{% macro default() %}
+  <i class="fa fa-star fa-lg rating"></i>
```

Comment:
Selectors like `.fa.fa-star.fa-lg.rating` are a violation of the [BEM](http://getbem.com/introduction/) naming convention